test/unit/services/oooRequest.test.ts

```
@@ -0,0 +1,161 @@
+import sinon from "sinon";
+import cleanDb from "../../utils/cleanDb";
+import {
+    OOO_STATUS_ALREADY_EXIST,
+    REQUEST_ALREADY_PENDING,
+    USER_STATUS_NOT_FOUND,
+} from "../../../constants/requests";
```

ESLint

[error] Replace `⏎····OOO_STATUS_ALREADY_EXIST,⏎····REQUEST_ALREADY_PENDING,⏎····USER_STATUS_NOT_FOUND,⏎` with `·OOO_STATUS_ALREADY_EXIST,·REQUEST_ALREADY_PENDING,·USER_STATUS_NOT_FOUND·`

(prettier/prettier)